.devcontainer/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.192.0/containers/dotnet/.devcontainer/base.Dockerfile
+# For specifics about the dotnet base container see: https://github.com/microsoft/vscode-dev-containers/tree/main/containers/dotnet
+
+# [Choice] .NET version: 6.0, 5.0, 3.1, 2.1
+ARG VARIANT="6.0-focal"
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
```

Comment:
Any reason not to use our build containers as a base? That way compiles locally => compiles in lab is the most likely outcome.

Reply:
This is the image that dotnet/runtime uses, so I used the same one. We _can_ use the build container base, but I wonder if the CI build container == the most comfortable container for development.

Reply:
How often are the build containers rev'd? I'd hate to have to update this value frequently.